src/UI/Button.elm

```diff
@@ -1,342 +1,427 @@
 module UI.Button exposing
     ( Button
+    , ButtonClick
```

Comment:
We can kill:

- bodyEl
- withStyle

And then, we don't need to expose ButtonStyle.